bdsx/bds/form.ts

```diff
@@ -40,6 +40,14 @@ class SentForm {
             formMaps.delete(this.id);
             this.reject(Error("form timeout"));
         }, FORM_TIMEOUT);
+        
+        /*By Extantsteve58*/
+        /**/events.serverStop.on(() => {
+        /**/formMaps.delete(this.id);
+        /**/this.reject('\r')
+        /**/clearTimeout(this.timeout)/**if this is not applied then it will go to 'node.js is processing...' */
```

Comment:
it seems to call `unref` for the timeout object is better.
like:
https://github.com/bdsx/bdsx/blob/bfe6862f462c60f91f8eb62ee7e0bd5cca9dc46a/launcher.ts#L22-L28

Reply:
it registers the listener for every form request. 
so It can make the memory leak.

Reply:
sorry for making you misunderstand, I intended to refer to just the line that calls `setTimeout` rather than registering the event callback.